src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1018,16 +995,15 @@
   (let [csr-path (path-to-cert-request csrdir subject)]
     (if (fs/exists? csr-path)
       (if (fs/delete csr-path)
-        (let [msg (str "Deleted certificate request for " subject)]
```

Comment:
They also get returned in a map that seems like it might be used by liberator and maybe returned in the response?  I don't really understand the liberator stuff well enough to know for sure that that is true, might be worth just curling the endpoint and seeing what the response looks like, or looking at the API docs?  Or @nathanielksmith maybe you already navigated through the liberator stuff and determined that these maps we return here don't make it all the way to the http response?